### PR TITLE
🚧Update workflow to use org level scripts

### PR DESCRIPTION
**Pull request by kselena**
<h1 style="font-weight:bold" align="center">Feature Pull Request</h1>
<h2 style="font-weight:bold" align="center">📋Review Checklist📋</h2>

- [x] The **_head(source)_** branch for this pull request is a **_feature_** branch, with the correct naming syntax, in the following example:
  - A **_head(source)_** branch is the branch being merged into a target branch.
    <details closed><summary>Example</summary>

      ``` xml
      Syntax: feature/<issue-num>-<description>
      Example: feature/123-my-feature
      ```
    </details>

- [x] The **_head(source)_** branch for this pull request is created from a **_development_** branch with the name **_develop_**.

- [x] The **_base(target)_** branch for this pull request is a **_development_** branch with the name **_develop_**.
  - A **_base(target)_** branch is the branch that the **_head(source)_** branch is merging into.

💡For more information on branching, refer to the project's [branching documentation](https://github.com/KinsonDigital/.github/blob/master/docs/BranchingFullGitFlow.md).

- [x] The associated issue exists and is linked to this pull request.
  - One issue per pull request.

- [x] The pull request title matches the title of the associated issue.

- [x] The labels attached to this PR match the labels attached to the associated issue.

- [x] The assigned project matches the project of the associated issue.

- [x] The assigned milestone matches the milestone of the associated issue.

- [x] My code follows the coding style of this project.
  - The style is enforced by the **_.editorconfig_** files in the project and displayed as warnings.  If there is an edge case, reach out, and let's discuss it.

- [x] I have manually tested my code changes to the best of my ability.

- [x] All tests passed locally.
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
  - Tests might pass on the developer's machine but not necessarily on the code reviewer's machine.
  - Status checks run for this pull request when merging into the **_develop_** branch.  These status checks run every time a change is pushed to the pull request.  These checks validate version syntax, tagging, builds, unit tests, and more.

### CODE REVIEW

**Review by KinsonDigitalAdmin — CHANGES_REQUESTED** [review]
Missed updated of the  **_release.yml_** workflow.   Please update this workflow in the same way. 

> **Note** An extra step was added to the issue.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -68,14 +68,13 @@ jobs:
...
-          "${{ github.workspace }}/.github/cicd/validate-tag.ts" `
+          "${{ github.workspace }}https://raw.githubusercontent.com/KinsonDigital/Infrastructure/v3.0.3/cicd/validate-tag.ts`
           "${{ inputs.release_type }}" `
           "v${{ steps.get-version.outputs.version }}" `
           "${{ env.project-name }}";
 
     - name: Validate Release Notes
-      run: deno run --allow-read "${{ github.workspace }}/.github/cicd/validate-notes.ts" "${{ inputs.release_type }}" "v${{ steps.get-version.outputs.version }}";
-
+      run: deno run --allow-read "${{ github.workspace }}https://raw.githubusercontent.com/KinsonDigital/Infrastructure/v3.0.3/cicd/validate-notes.ts`
```
Comment (KinsonDigitalAdmin) [inline .github/workflows/release.yml:77]: Please fix the `deno run` command.  You added the URL in place of the file path but you removed all of the other parts of the command which is the arguments being sent into the script.

> **Note** Refer to the _**Validate Tag**_ step before this step for reference as a guide.

```suggestion
      run: deno run `
        --allow-read `
        "${{ github.workspace }}https://raw.githubusercontent.com/KinsonDigital/Infrastructure/v3.0.3/cicd/validate-notes.ts `
        "${{ inputs.release_type }}" `
        "v${{ steps.get-version.outputs.version }}";
```

**Review by KinsonDigitalAdmin — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -68,14 +68,19 @@ jobs:
...
     - name: Validate Release Notes
-      run: deno run --allow-read "${{ github.workspace }}/.github/cicd/validate-notes.ts" "${{ inputs.release_type }}" "v${{ steps.get-version.outputs.version }}";
-
+      run: 
+        deno run `
+        --allow-read `
+          "${{ github.workspace }}https://raw.githubusercontent.com/KinsonDigital/Infrastructure/v3.0.3/cicd/validate-notes.ts `
+          "${{ inputs.release_type }}" `
+          "v${{ steps.get-version.outputs.version }}";
+
```
Comment (KinsonDigitalAdmin) [inline .github/workflows/release.yml:83]: You forgot the pipe character that makes this a multi-line shell script.  It is always useful to use other pieces of a workflow as a guide on how to do something to help you out.

> **Note** No empty lines below the `run:  |` and the script code.  You _**CAN**_ have empty lines but it is just considered empty shell script lines and it is not clean.

Please fix.

```suggestion
      run:  |
        deno run `
        --allow-read `
          "${{ github.workspace }}https://raw.githubusercontent.com/KinsonDigital/Infrastructure/v3.0.3/cicd/validate-notes.ts `
          "${{ inputs.release_type }}" `
          "v${{ steps.get-version.outputs.version }}";       
```

**Example:**
image

**Review by KinsonDigitalAdmin — APPROVED** [review]

File: .github/workflows/version-notes-status-check.yml [hunk 3]
```diff
@@ -82,6 +82,6 @@ jobs:
         deno `
           run `
           --allow-read `
-          "${{ github.workspace }}/.github/cicd/validate-notes.ts" `
+          "${{ github.workspace }}https://raw.githubusercontent.com/KinsonDigital/Infrastructure/v3.0.3/cicd/validate-notes.ts`
```
Comment (KinsonDigitalAdmin) [inline .github/workflows/version-notes-status-check.yml:85]: Remove the `${{ github.workspace }}` variable.  This is a path and for paths. But now we are using URLs.
Don't forget the double quote in the beginning.  You removed the one on the right end when you entered the URL only leaving one behind without a matching pair.

```suggestion
          https://raw.githubusercontent.com/KinsonDigital/Infrastructure/v3.0.3/cicd/validate-notes.ts`
```

File: .github/workflows/version-notes-status-check.yml [hunk 4]
```diff
@@ -72,7 +72,7 @@ jobs:
         deno run `
           --allow-run `
           --allow-net `
-          "${{ github.workspace }}/.github/cicd/validate-tag.ts" `
+          "${{ github.workspace }}https://raw.githubusercontent.com/KinsonDigital/Infrastructure/v3.0.3/cicd/validate-tag.ts`
```
Comment (KinsonDigitalAdmin) [inline .github/workflows/version-notes-status-check.yml:75]: Remove the `${{ github.workspace }}` variable.  This is a path and for paths. But now we are using URLs.
Don't forget the double quote in the beginning.  You removed the one on the right end when you entered the URL only leaving one behind without a matching pair.

```suggestion
          https://raw.githubusercontent.com/KinsonDigital/Infrastructure/v3.0.3/cicd/validate-tag.ts`
```

File: .github/workflows/release.yml [hunk 5]
```diff
@@ -68,14 +68,19 @@ jobs:
...
           "v${{ steps.get-version.outputs.version }}" `
           "${{ env.project-name }}";
 
     - name: Validate Release Notes
-      run: deno run --allow-read "${{ github.workspace }}/.github/cicd/validate-notes.ts" "${{ inputs.release_type }}" "v${{ steps.get-version.outputs.version }}";
-
+      run: |
+        deno run `
+          --allow-read `
+          "${{ github.workspace }}https://raw.githubusercontent.com/KinsonDigital/Infrastructure/v3.0.3/cicd/validate-notes.ts `
```
Comment (KinsonDigitalAdmin) [inline .github/workflows/release.yml:80]: Remove the `${{ github.workspace }}` variable.  This is a path and for paths. But now we are using URLs.
Don't forget the double quote in the beginning.  You removed the one on the right end when you entered the URL only leaving one behind without a matching pair.

```suggestion
          https://raw.githubusercontent.com/KinsonDigital/Infrastructure/v3.0.3/cicd/validate-notes.ts `
```

File: .github/workflows/release.yml [hunk 6]
```diff
@@ -68,14 +68,19 @@ jobs:
         deno run `
           --allow-run `
           --allow-net `
-          "${{ github.workspace }}/.github/cicd/validate-tag.ts" `
+          "${{ github.workspace }}https://raw.githubusercontent.com/KinsonDigital/Infrastructure/v3.0.3/cicd/validate-tag.ts`
```
Comment (KinsonDigitalAdmin) [inline .github/workflows/release.yml:71]: Remove the `${{ github.workspace }}` variable.  This is a path and for paths. But now we are using URLs.
Don't forget the double quote in the beginning.  You removed the one on the right end when you entered the URL only leaving one behind without a matching pair.

```suggestion
          https://raw.githubusercontent.com/KinsonDigital/Infrastructure/v3.0.3/cicd/validate-tag.ts`
```

**Review by KinsonDigitalAdmin — CHANGES_REQUESTED** [review]

**Review by KinsonDigitalAdmin — APPROVED** [review]
